source/hud/blocklyButton.js

```diff
@@ -22,8 +22,16 @@ this.onClick = function() {
     var hud = this.parent;
     hud.lastBlocklyNodeID = hud.lastBlocklyNodeID || this.scene.player.rover.id;
     if ( hud.lastBlocklyNodeID !== undefined ) {
+        
+        var blocklyNode = blocklyNodes[ hud.lastBlocklyNodeID ];
+
+        if ( blocklyNode && Blockly.mainWorkspace ) {
+            Blockly.mainWorkspace.maxBlocks = blocklyNode.ramMax;
+        } else {
+            console.log("Either blocklyNode or Blockly.mainWorkspace are missing!");
+        }
```

Comment:
I think this is getting overridden by the `satProperty` in `blockly.js`? If I add the following to `index.js`:

```
case "blockly_activeNodeID":
    console.log("Active node ID sat in index.js!"); 
    break;
```

I never see the above message printed.